content/reference/endpoints/los_records.md

```diff
@@ -12,25 +12,25 @@
   <div class="tab-pane active" id="public" markdown="1">
 Name                 | Type    | Read/Write | Description
 ---------------------|---------|------------|------------
-id                   | Integer | Read       | Uniq identifier.
+id                   | Integer | Read       | Unique identifier.
 ---------------------|---------|------------|------------
 kind                 | String  | Read       | LOS record's kind. For possible values visit the [enums section](/reference/enums#los-kinds).
-day                  | [Date](/reference/enums#formats) | Read       | Day for which LOS record is valid.
-min_occupancy        | Integer | Read       | Minimum occupancy for which the LOS record is valid.
-max_occupancy        | Integer | Read       | Maximum occupancy for which the LOS record is valid.
-original_currency    | String  | Read       | Currency in which the rates are originating from.<br>The list of codes can be found in the [enums section](/reference/enums#currencies).
-currency             | String  | Read       | Currency in which the rates are in the response.<br>The list of codes can be found in the [enums section](/reference/enums#currencies).
-exchange_rate        | [Decimal](/reference/enums#formats) | Read       | The rate used to convert from the `original_currency` to `currency`.
+day                  | [Date](/reference/enums#formats) | Read       | Day for which this LOS record is valid.
+min_occupancy        | Integer | Read       | Minimum occupancy for which this LOS record is valid.
+max_occupancy        | Integer | Read       | Maximum occupancy for which this LOS record is valid.
+original_currency    | String  | Read       | Original currency that LOS record was generated with (useful when using `exchange_to_currency` query param).<br>The list of codes can be found in the [enums section](/reference/enums#currencies).
```

Comment:
I would even add write it that way `useful when using `exchange_to_currency` query param` -> `useful when using `exchange_to_currency` query param as a reference currency`